src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditContainer.tsx

```diff
@@ -122,14 +123,14 @@ const WorkPackageEditContainer: React.FC<WorkPackageEditContainerProps> = ({ wor
     return `${date.getFullYear().toString()}-${month}-${day}`;
   };
   const onSubmit = async (data: WorkPackageEditFormPayload) => {
-    const { name, projectLead, projectManager, startDate, duration, crId, blockedBy, stage } = data;
+    const { name, startDate, duration, crId, blockedBy, stage } = data;
     const expectedActivities = mapBulletsToPayload(data.expectedActivities);
     const deliverables = mapBulletsToPayload(data.deliverables);
 
     try {
       const payload = {
-        projectLead,
-        projectManager,
+        projectLead: leadId ? parseInt(leadId) : leadId,
```

Comment:
instead of setting it to leadId when there is none, set it to undefined, it becomes the same thing but for readabliltiy